src/dialects/postgres/query/compiler.js

```diff
@@ -75,6 +75,20 @@ assign(QueryCompiler_PG.prototype, {
     return 'for share';
   },
 
+  // Compiles a `listTables` query
+  listTables() {
+    return {
+      sql: 'select table_name from information_schema.tables' +
```

Comment:
Here I believe we'll want `schemaName` option somehow, since its pretty common not to be using `public`. A good start might be `where table_schema = current_schema()`, but also have an option.

I'd also recommend `table_catalog = current_catalog()` over the explicit binding.

https://www.postgresql.org/docs/current/static/functions-info.html